### PR TITLE
Fixing builds, upgraded to angular 1.4.1

### PR DESCRIPTION
**Pull request by JWGmeligMeyling**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,30 +2,26 @@
...
     "angular-audio": "~1.6.2",
-    "angular-timer": "~1.3.1",
-    "angular-ui-router": "~0.2.14",
+    "angular-timer": "~1.3.3",
+    "angular-ui-router": "~0.2.15",
     "ui-router-extras": "~0.0.13"
   },
   "devDependencies": {
-    "angular-mocks": "^1.3.0",
-    "angular-gravatar": "~0.3.1",
```
Comment (JWGmeligMeyling) [inline bower.json]: Gravatar is not used anymore as avatar provider, we use the avatars provided by soundcloud.

File: bower.json [hunk 2]
```diff
@@ -2,30 +2,26 @@
...
-    "angular-ui-router": "~0.2.14",
```
Comment (JWGmeligMeyling) [inline bower.json]: ui-router already is a dependency and can be removed as dev dependency

File: bower.json [hunk 3]
```diff
@@ -2,30 +2,26 @@
...
   "dependencies": {
-    "angular": "^1.3.0",
+    "angular": "1.4.1",
     "bootstrap-sass-official": "^3.2.0",
-    "angular-animate": "^1.3.0",
-    "angular-aria": "^1.3.0",
-    "angular-cookies": "~1.3.15",
-    "angular-messages": "^1.3.0",
-    "angular-resource": "^1.3.0",
-    "angular-route": "^1.3.0",
```
Comment (JWGmeligMeyling) [inline bower.json]: We have switched from ngRoute to ui-router, so this dependency is not needed anymore.

File: bower.json [hunk 4]
```diff
@@ -2,30 +2,26 @@
...
-    "angular-gravatar": "~0.3.1",
-    "angular-ui-router": "~0.2.14",
-    "ui-router-extras": "~0.0.13"
+    "angular-mocks": "^1.4.1"
   },
   "appPath": "app",
   "moduleName": "moodCatApp",
   "resolutions": {
-    "angular": "1.4.0"
+    "angular": "1.4.1"
```
Comment (JWGmeligMeyling) [inline bower.json]: Resolving to 1.4.1 now.

**Comment by JeremybellEU** [comment]
Working as intended. Merging!

**Pull request merged**